src/frontend/tests/features/playwright.config.ts

```diff
@@ -0,0 +1,9 @@
+import { PlaywrightTestConfig } from "@playwright/test";
+const config: PlaywrightTestConfig = {
+  timeout: 60000,
+  use: {
+    screenshot: "only-on-failure",
+    ignoreHTTPSErrors: true,
```

Comment:
This tells playwright to automatically take a screenshot on test failure, and configures it to play along with the self-signed certs we use in local dev.